benchmark/single-source/SetTests.swift

```diff
@@ -58,10 +58,15 @@ let setQ: Set<Int> = {
 public let SetTests = [
   // Mnemonic: number after name is percentage of common elements in input sets.
   BenchmarkInfo(
-    name: "Set.Empty.IsSubsetInt0",
+    name: "Set.isSubset.Empty.Int0",
```

Comment:
Minor nitpick (it's not important, but since we're changing these names anyway...): 

`Int0` and `Box0` are a little misleading in the `Empty` cases -- `Int`/`Box` would suffice. The number specifies the overlap percentage, but `Empty` is a more interesting signal here.

Reply:
I originally left it in to indicate the overlap was 0, but it in this case, given it IS a subset, that is outright misleading. I'll remove the `0`s from `Empty` names...